types/field_type_test.go

```diff
@@ -199,6 +203,25 @@ func (s *testFieldTypeSuite) TestDefaultTypeForValue(c *C) {
 	}
 }
 
+// BenchmarkDefaultTypeForValue
+// run with: GO111MODULE=on go test -bench=BenchmarkDefaultTypeForValue
+var r uint64 = 0xdeadbeef
```

Comment:
Extract these newly added vars and function to a new file `types/benchmark_test.go`

Reply:
Brilliant 🀄️